### DISCUSSION TITLE
Announcing Ascent

### DISCUSSION THREAD

**Discussion by DominusKelvin** [sailscastshq/boring-stack#121] [Announcements]
TBJS (The Boring JavaScript Stack) aims to handle essential yet boring tasks in full-stack JavaScript apps, letting developers focus on shipping their products.

Mellow was the first step, offering built-in authentication, email, Google OAuth via Sails Wish, and basic user profile management.

Ascent takes it further by providing a complete, secure, free, open-source JavaScript SaaS template powered by Inertia.js. It includes:

1. Waitlist
2. Authentication
3. OAuth and Magic link login
4. 2FA
5. Subscription payments (Lemon Squeezy, Stripe, etc, via Sails Pay)
6. Content blog (Sails Content)
7. Transactional email (Sails Mail)
8. Multi-tenancy/Teams
9. Admin dashboard
10. And more

Ascent will be available in React, Vue, and Svelte (React first) and will be developed live, starting November 1, 2024, on YouTube, X, and Twitch.

[Check out the post on X](https://x.com/Dominus_Kelvin/status/1852311756884054438)

**Comment by lennyAiko** [comment]
Now this is amazing! 🔥
